experiments/AMIP/Manifest.toml

```diff
@@ -27,9 +27,9 @@ weakdeps = ["ChainRulesCore", "Test"]
     AbstractFFTsTestExt = "Test"
 
```

Comment:
I don't think the experiments/AMIP Manifest should have to change for this PR. Can you try resetting this file to what it is on main and see if that still runs okay?